resources/views/index.blade.php

```diff
@@ -14,6 +14,7 @@
     <link rel="stylesheet" type="text/css" href="angular/account/register.css">
     <link rel="stylesheet" type="text/css" href="angular/about/about.css">
     <link rel="stylesheet" type="text/css" href="angular/admin/privacy.css">
+    <link rel="stylesheet" type="text/css" href="angular/home/welcome.css">
```

Comment:
This weekend I'll create a branch with better Gulp examples so you can see how it works if you've not used it. Will clean this right up.